pygeoapi/process/manager/__init__.py

```diff
@@ -1,8 +1,10 @@
 # =================================================================
```

Comment:
Maybe this is missing a rebase from a previous merge @ricardogsilva, can you have a look?